packages/radio-group/src/Radio/Radio.tsx

```diff
@@ -71,6 +74,7 @@ function Radio({
         className={cx(
           labelBaseStyle,
           {
+            [labelDisabledStyle[theme]]: disabled,
```

Comment:
Can you use the disabled prop instead of adding CSS? It looks like the colors are the same.

Reply:
Good catch